assets/elm/Terms/Models.elm

```diff
@@ -8,6 +8,20 @@ type alias Terms =
     }
 
 
+type Workflow
+    = Combined Term
+    | WholeName Term
+    | SplitName Term
+
+
+type alias Term =
+    { value : String
+    , unique : Bool
```

Comment:
I try to avoid `Bool` as aggressively as possible; when reading where it's used, a reference to the name is necessary to understand what it's doing (e.g. `Term "Foo" True "desc" "google.com"` - what does `True` mean here?). Instead, I'll create a sum type to capture the concept, e.g. `type UniqueTerm = MustBeUnique | AllowsDuplicates`; with that, it's obvious both during construction and branches what the possible states are.